src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1379,4 +1379,7 @@ This version used in: {2}</value>
   <data name="Formatting_document" xml:space="preserve">
     <value>Formatting document</value>
   </data>
+  <data name="Modifying_source_file_will_prevent_the_debug_session_from_continuing_due_to_internal_error" xml:space="preserve">
```

Comment:
Heads up that if this gets discussed for a 15.8 backport the loc changes might be problematic.

Reply:
I know. That's why i did not include it in the targeted 15.8 fix.